File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Location.java
package com.tistory.black_jin0427.myandroidarchitecture.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by me on 7/25/17.
 */

public class Location implements Serializable {

    @SerializedName("street") public String street;

    @SerializedName("city") public String city;

    @SerializedName("state") public String state;

    @SerializedName("postcode") public String postcode;

}

File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Location.kt
package com.tistory.black_jin0427.myandroidarchitecture.api.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class Location(

    @SerializedName("street")
    var street: String?,

    @SerializedName("city")
    var city: String?,

    @SerializedName("state")
    var state: String?,

    @SerializedName("postcode")
    var postcode: String?

) : Serializable
